src/midi.js

```diff
@@ -22,19 +29,26 @@ const midi = (notes, fileName) => {
 		// only the first noteOn (or noteOff) needs the complete arity of the function call
 		// subsequent calls need only the first 2 args (channel and note)
 		if (noteObj.note) {
+			var note = noteObj.note[0];
+			//Get the octave identifier (2 for a2, 5 for e5)
+			var oct = parseInt(note.slice(1,note.length));
+			//Parse the octave into an integer
+			oct+=transposition;
```

Comment:
Add a space on the either side of `+=`.
`oct += transposition;` instead of `oct+=transposition;`